src/app/contexts/UserContext/cookies/index.js

```diff
@@ -6,12 +6,21 @@ export const getCookiePolicy = () => {
   return Cookie.get(POLICY_COOKIE) || '000';
 };
 
+export const getPreferredVariantCookieName = service => `ckps_${service}`;
```

Comment:
Is this exported function used anywhere, or could you just set the cookie to `ckps_${service}` below (in the same way as `VARIANT_COOKIE` is set in `getPreferredVariant`?

Reply:
`getPreferredVariantCookieName` was not being used elsewhere. Updated `setPreferredVariantCookie` to determine the variant cookie name as suggested